ytools/yast2/create_maintenance_branch

```diff
@@ -52,8 +52,9 @@ def modify_rakefile
   line_index = lines.index {|l| l =~ /#{submit_to}/ }
   if line_index
     lines[line_index] = new_line
-  else
-    lines.insert(2, new_line, "\n")
+  else # line is not there yet, so place it below require line
+    line_index = lines.index {|l| l =~ /require.*yast\/rake/ }
+    lines.insert(lines_index, "\n", new_line)
```

Comment:
not tested, as it creates branch, which I do not like to do, but will try it now, when I remove some accidentally created branches like one in yast-rake itself